docs/openhabian.md

```diff
@@ -239,6 +239,18 @@ Here are the passwords in question, their default value and the way to change th
 * Karaf remote console (e.g. "openhab:habopen"): Change via the openHABian menu
 * Nginx reverse proxy login (no default): Change via the openHABian menu, please see [here](http://docs.openhab.org/installation/security.html#adding-or-removing-users) for more
 
+## Optional Components
+
+openHABian comes with a number of additional configs that allow you to quickly install home automation related software.
+
+* [frontail](https://github.com/mthenw/frontail) - openHAB Log Viewer accessible from [http://openHABianPi:9001/](http://openHABianPi:9001/)
+* [KNXd](http://michlstechblog.info/blog/raspberry-pi-eibknx-ip-gateway-and-router-with-knxd) - KNX daemon running at `224.0.23.12:3671/UDP`
+* [Homegear](https://www.homegear.eu/index.php/Main_Page) - Homematic control unit emulation
+* [Eclipse Mosquitto](http://mosquitto.org) - Open Source MQTT v3.1/v3.1.1 Broker
+* [OWServer](http://owfs.org/index.php?page=owserver_protocol) - 1wire control system
+* [Grafana](https://community.openhab.org/t/influxdb-grafana-persistence-and-graphing/13761/1) - persistence and graphing available from [http://openHABianPi:3000/](http://openHABianPi:3000/)
+* [Node-RED](https://nodered.org/) - Flow-based programming for the Internet of Things with preinstalled [openHAB2](https://flows.nodered.org/node/node-red-contrib-openhab2) and [BigTimer](https://flows.nodered.org/node/node-red-contrib-bigtimer) addons. Accessible from [http://openHABianPi:1880/](http://openHABianPi:1880/)
+
```

Comment:
You probably wanted to remove this at the old location?
I'm not against moving these down here but they may get not as much attention down here, so better put a reference to here up there?